zmq-async.opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "anders@fugmann.net"
+authors: ["Rudi Grinberg"]
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+dev-repo: "git@github.com/issuu/ocaml-zmq.git"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "zmq"
+  "jbuilder" {build}
```

Comment:
Are the build dependencies really needed for compiling zmq-async. We do already depend on zmq, which needed the packages when building.

Reply:
Yes, it's better not to rely on dependencies being pulled transitively. It can cause needless breakage when your dependencies change what they depend on or how they're built. In this particular case, our zmq-async package would work even with older versions of zmq that were built using oasis.